osu.Game.Benchmarks/osu.Game.Benchmarks.csproj

```
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <OutputType>Exe</OutputType>
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="BenchmarkDotNet" Version="0.12.0" />
+    <PackageReference Include="nunit" Version="3.12.0" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.16.0" />
```

```suggestion
    <PackageReference Include="NUnit3TestAdapter" Version="3.15.1" />
```

See: https://youtrack.jetbrains.com/issue/RIDER-37769 . Hinders our workflow for now.